dev-db/erd/erd-0.2.1.0.ebuild

```
@@ -29,3 +29,10 @@ DEPEND="${RDEPEND}
 		>=dev-haskell/tasty-hunit-0.9 )
 "
 PATCHES=("${FILESDIR}/${PN}-0.2.1.0-ghc-9.patch")
+
+src_prepare() {
+	default
+
+	cabal_chdeps \
+		'bytestring == 0.10.*' 'bytestring == 0.11.*'
+}
```

```suggestion
CABAL_CHDEPS=(
	'bytestring == 0.10.*' 'bytestring == 0.11.*'
)
```

`cabal_chdeps` was deprecated in 0bcd1a1e80155c98ec3a2304e4b6cf9e763b9d64. Please use `CABAL_CHDEPS`.

`haskell-cabal_src_prepare` does quite a bit now, so it isn't a good idea to override `src_prepare` like this anymore.

If you absolutely have to, replace `default` with `haskell-cabal_src_prepare`, but `CABAL_CHDEPS` was added to make this common use-case more convenient.

---

I'm also unsure why you would preclude `bytestring == 0.10.*` here. If it works with both `bytestring == 0.10.*` and `bytestring == 0.11.*`, then it would make sense just to remove the upper bound like so:

```ebuild
CABAL_CHDEPS=(
	'bytestring == 0.10.*' 'bytestring >= 0.10'
)